fastcrawler/core/process.py

```diff
@@ -31,7 +31,7 @@ def __init__(
         else:
             self.task = Task(
                 start_cond=cond or "every 1 second",
-                name=spider.__class__.__name__ + str(uuid4()),
+                name=f"{uuid4()}@{spider.__class__.__name__}",
```

Comment:
this needs lots of comments and examples

Reply:
Will add doc src in future.